lib/primitive/lib/Cardano/Wallet/Primitive/Types/Tx/Constraints.hs

```diff
@@ -129,22 +140,13 @@ newtype TxSize = TxSize { unTxSize :: Natural }
     deriving stock (Eq, Ord, Generic)
     deriving Show via (Quiet TxSize)
     deriving Num via Natural
+    deriving (Commutative, Semigroup, Monoid, MonoidNull) via Sum Natural
+    deriving (LeftReductive, RightReductive, Reductive) via Sum Natural
+    deriving (LeftGCDMonoid, RightGCDMonoid, GCDMonoid) via Sum Natural
+    deriving (OverlappingGCDMonoid, Monus) via Sum Natural
```

Comment:
We really only need a `Monus` instance (which gives us the `<\>` monus subtraction operator), but `Monus` has a few superclasses. Luckily, they can all be derived.